espnet/nets/scorer_interface.py

```diff
@@ -100,7 +101,19 @@ def batch_score(
                 and next state list for ys.
 
         """
-        raise NotImplementedError
+        warnings.warn(
+            "this {} batch score is implemented throught for loop not parallel".format(
```

Comment:
Typo and some English correction

"this {} batch score is implemented throught for loop not parallel"
->
"this {} batch score is implemented through for loop and not parallelized"

Reply:
lol, might I leave English environment for a too long time, I will fix it quickly.